### DISCUSSION TITLE
Twint?

### DISCUSSION THREAD

**Discussion by Foolian2** [bisq-network/bisq#7223] [Ideas]
For all my fellow people of Switzerland, I would like to see to add Twint as an account method. Thus  is is very popular in Switzerland.
https://www.twint.ch/

**Comment by alvasw** [comment]
Hi @Foolian2!

Yes, we can add Twint to Bisq as a payment method. Could you create the request here ([1](https://github.com/bisq-network/growth/issues/new?assignees=&labels=re%3Apayment-method&template=add_payment_method.md&title=)) by filling out the required information (Data requirements - What information must a user provide to a counterparty to accept a payment?, Duration, ...)?
Thank you!

[1] https://github.com/bisq-network/growth/issues/new?assignees=&labels=re%3Apayment-method&template=add_payment_method.md&title=
